tasks/compile-example.js

```diff
@@ -273,7 +273,7 @@ module.exports = function(config, indexPath, updateTimestamp) {
         compileTemplate(stream, example, args, {
           template: previewTemplate,
           targetPath: example.targetPreviewPath(),
-          postProcessors: [replaceAmpAdRuntime, replaceAmpStoryRuntime],
+          postProcessors: [replaceAmpAdRuntime, replaceAmpStoryRuntime, replaceAmpHtmlEmailRuntimeAddViewport],
```

Comment:
Can you do me a favour and extract all post processors into a list, this way it becomes easier to add more. Thanks!

Reply:
done